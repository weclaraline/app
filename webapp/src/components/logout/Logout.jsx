import React from "react";
import { GoogleLogout } from "react-google-login";

const clientId =
  "903919404222-hb622tgshpbbj1fudcp4hlgkuc9fh027.apps.googleusercontent.com";

function Logout() {
  const onSucces = (response) => {
    alert("Logut made succesfully");
    console.log(response);
  };

  return (
    <div>
      <GoogleLogout
        clientId={clientId}
        buttonText="logout"
        onLogoutSuccess={onSucces}
      ></GoogleLogout>
    </div>
  );
}

export default Logout;
