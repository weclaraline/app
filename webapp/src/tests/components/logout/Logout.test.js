import '../../testing_utils';
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Logout from "../../../components/logout/Logout";

it("Renders original content", ()=> {
    const tree = shallow(<Logout />);
    expect(toJson(tree)).toMatchSnapshot();
});