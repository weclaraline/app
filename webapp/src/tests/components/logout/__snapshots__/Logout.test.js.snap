// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders original content 1`] = `
<div>
  <v
    buttonText="Cerrar sesiÃ³n"
    clientId="903919404222-hb622tgshpbbj1fudcp4hlgkuc9fh027.apps.googleusercontent.com"
    disabledStyle={
      Object {
        "opacity": 0.6,
      }
    }
    icon={true}
    jsSrc="https://apis.google.com/js/api.js"
    onLogoutSuccess={[Function]}
    tag="button"
    theme="light"
    type="button"
  />
</div>
`;
